Documentation/gettingstarted/k8s-install-kops.rst

```diff
@@ -128,7 +128,7 @@ You may be prompted to create a ssh public-private key pair.
 
 (Please see :ref:`appendix_kops`)
 
-.. include:: k8s-install-connectivity-test.rst
+.. include:: k8s-install-validate.rst
```

Comment:
To reviewers: will `kops` allow for usage of Cilium CLI? If not, then we should revert to manually linking to
```suggestion
.. include:: k8s-install-connectivity-test.rst
```

Reply:
Discussed on public Slack: `kops` works with the CLI.
https://cilium.slack.com/archives/C2B917YHE/p1621609041057000?thread_ts=1621608612.056800&cid=C2B917YHE